lib/view_component/engine.rb

```diff
@@ -8,6 +8,10 @@ class Engine < Rails::Engine # :nodoc:
     config.view_component = ActiveSupport::OrderedOptions.new
     config.view_component.preview_paths ||= []
 
+    rake_tasks do
+      load "view_component/rails/tasks/view_component.rake"
```

Comment:
Perhaps we could add a CI task that runs `rails stats`?

Reply:
I took some time to search and I found a way to add a test for that task :)